src/apps/profiles/models.py

```diff
@@ -38,7 +40,19 @@ class User(ChaHubSaveMixin, AbstractBaseUser, PermissionsMixin):
 
     # Any User Attributes
     username = models.CharField(max_length=50, unique=True)
-    email = models.CharField(max_length=200, unique=True, null=True, blank=True)
+    slug = models.SlugField(max_length=50, default='', unique=True)
+    photo = models.ImageField(upload_to=PathWrapper('profile_photos'), null=True, blank=True)
+    email = models.EmailField(max_length=200, unique=True, null=True, blank=True)
+    display_name = models.CharField(max_length=50, unique=True, null=True, blank=True)
```

Comment:
In my opinion, we don't need a username, email, first, last, and display name for a single user. Probably we could do without `display_name`. Do you know why we would need to use both username and display name?

Reply:
If this was requested, disregard this comment.

Reply:
All of these were in the spec. I would agree it's a bit redundant. Maybe we should just have `display_name` and `username`? The user could still enter their full name in the display name.